src/components/UploadPopUp/UploadPopUp.jsx

```diff
@@ -34,15 +55,15 @@ export default function UploadPopUp({ open, close }) {
         <h1> Upload Timetable</h1>
       </div>
 
-      <form onSubmit={handleSubmit}>
+      <form onSubmit={handleSubmit} >
         <div className={styles.uploadContent}>
           <Button
             component="label"
             variant="contained"
             sx={{ maxHeight: '40px', minWidth: '120px', minHeight: '40px' }}>
             <UploadIcon />
             Upload File
-            <input type="file" onChange={handleOnChange} hidden accept=".ics" id="fileUpload"/>
+            <input type="file" onChange={handleOnChange} hidden multiple="multiple" accept=".ics" id="fileUpload"/>
```

Comment:
If I'm understanding the codebase correctly, I don't think file upload should allow multiple inputs. #141 clearly mentions that only one file should be uploaded as it's a timetable. @christopher-alba, could you confirm this?

```suggestion
            <input type="file" onChange={handleOnChange} hidden accept=".ics" id="fileUpload"/>
```